CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+# 4.8.0 / 2020-12-11
+* [FEATURE] Add `env` and `log_to_console` options [#759][] [@albertvaka][] and [#757][] [@nilskuehme][]
```

Comment:
#757 is the release PR for 4.7.2, is that the one you wanted to put here?

Reply:
Nope :)

Reply:
Fixed